crates/criticaltrust/src/errors.rs

```diff
@@ -23,6 +26,14 @@ pub enum Error {
     InvalidKey(String),
     #[error("unsupported key")]
     UnsupportedKey,
+    #[error("verification failed because the signatures expired")]
+    SignaturesExpired,
+    #[error("revocation info is expired")]
+    RevocationSignatureExpired,
+    #[error("failed to verify signed data because payload is revoked")]
+    ContentRevoked,
```

Comment:
Can we inform the user which package/release so that if they contact us we can easily find it?

Reply:
Yes! Does this look better? I feel its repetitive.

```
error: failed to install package 'rustc-x86_64-unknown-linux-gnu' because it has been revoked
  caused by: failed to verify signed package 'rustc-x86_64-unknown-linux-gnu' because content is revoked
```

Reply:
Changed a few things to 

```
error: failed to install package 'rustc-x86_64-unknown-linux-gnu'
  caused by: failed to verify signed package 'rustc-x86_64-unknown-linux-gnu' because content is revoked
```